src/components/Form/ProjectsDataDatePicker.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import DateView from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+
+import { Field, ErrorMessage } from "formik";
+import TextError from "./TextError";
+
+
+function ProjectsDataDatePicker(props) {
+    const { label, name, ...rest } = props;
+
+    return (
+        <div className="form-group-project">
+            <label className="form-label-project" htmlFor={name}>{label}</label>
+            <div className="form-control-project">
+                <Field name={name}>
+                    {
+                        ({ form, field }) => {
+                            const { setFieldValue } = form
+                            const { value } = field
+                            return (
+                                <DateView
+                                    className="focus:outline-none"
+                                    id={name} {...field} {...rest}
+                                    selected={value}
```

Comment:
Change to selected={(value && new Date(value)) || null}

Reply:
Resolved